test/e2e/components/vl-select.js

```diff
@@ -35,47 +42,109 @@ class VlSelect extends VlElement {
             return this.findElements(By.css('option'));
         }
     }
-    
+
     async _mapDressedOptions(options) {
         return Promise.all(options.map(async (selectItem) => {
             const text = await selectItem.getText();
-            return {text: text, webElement: selectItem};
+            const webElement = await selectItem;
+            return { text: text, webElement: webElement };
+        }));
+    }
+
+    async _getWebelementMap() {
+        await this._openDressedDropdown();
+        const selectItems = await this._getOptions();
+        return this._mapDressedOptions(selectItems);
+    }
+
+    async _getValue(element) {
+        if((await this.isDressed())){
+            return element.getAttribute('data-value');
+        } else {
+            return element.getAttribute('value');
+        }
+    }
+
+    async _mapValues(options) {
+        return Promise.all(options.map(async (option) => {
+            const value = await this._getValue(option);
+            return { webElement: option, value: value }
+        }));
+    }
+
+    async _mapVisibleText(options) {
```

Comment:
Waarom is het nodig om alle spaties/tabs te vervangen met spaties en dan te trimmen?

Reply:
Als ik de textContents opvraag van bijvoorbeeld een option of een div in een dressed dropdown, dan komen er whitespace karakters mee, bijvoorbeeld `'\n            Label one\n          '` Deze whitespaces worden door de regex vervangen door 1 spatie, die vervolgens dan getrimmed worden.